src/maincurrencydialog.cpp

```diff
@@ -939,11 +929,14 @@ bool mmMainCurrencyDialog::GetOnlineHistory(std::map<wxDateTime, double> &histor
         if (quotes_closed[i].IsFloat())
         {
             double rate = quotes_closed[i].GetFloat();
-            history[time] = rate;
-            if (first_date > time)
+            if (rate != 1) // Skip rates = 1 (Yahoo returns 1 with invalid Symbols)
```

Comment:
What with following downloaded history values:
1.11
1.09
1.03
1.00
0.98
0.99
1.00
1.01
?
Two of them ("1.00") will be missing?

Reply:
I think it's nearly impossibile that two currencies has exact 1:1 conversion rate

Reply:
There are technical currency clones (EUR vs WIR EUR etc.) and some currencies can have similar rates for political reasons. I can imagine ratio oscillating around 1:1